tests/unit/amo/reducers/test_home.js

```diff
@@ -543,6 +513,45 @@ describe(__filename, () => {
     });
   });
 
+  describe('createInternalHeroCallToAction', () => {
+    it('creates an internal representation of the call to action property', () => {
+      const secondaryShelf = createSecondaryHeroShelf();
+
+      expect(
+        createInternalHeroCallToAction(secondaryShelf.cta, lang),
+      ).toMatchObject({
```

Comment:
https://github.com/mozilla/addons-frontend/pull/10086/commits/445f4a83d528cad2a53b9a621f9166423a0a19bd okay good to know - because the other existing tests used `toMatchObject` I assumed it was a special case of `toEqual` that worked better for objects.